versioned_docs/version-1.0/tutorial/02-hello-world.md

```diff
@@ -104,7 +95,9 @@ You will start by using a smart contract that contains a public function that re
 
 Like most other blockchains, the programming model in Flow is centered around accounts and transactions.
 All state that persists permanently is stored in [accounts](../language/accounts)
-and all accounts have the same core functionality. (users, smart contracts, data storage)
+and all accounts have the same core functionality. (users, smart contracts, data storage).
```

Comment:
prob not, but should we mention EOAs

Reply:
I'm talking about fundamental functionality. EOAs are a 2nd order construct in Cadence for flow-evm so don't want to mention them yet.